src/helpers.js

```diff
@@ -54,7 +54,11 @@ const helpers = {
    */
   getTxType(tx) {
     if (this.isBlock(tx)) {
-      return 'Block';
+      if (tx.version === BLOCK_VERSION) {
+        return 'Block';
+      } else if (tx.version === MERGED_MINED_BLOCK_VERSION) {
+        return 'Merged mined block';
```

Comment:
I'd say "Merged Mining Block" or "MM Block". What do you think?

Reply:
Changed to "Merged Mining Block"